torch/quantization/observer.py

```diff
@@ -656,8 +668,10 @@ def forward(self, x_orig):
         else:
             min_vals = min_vals + self.averaging_constant * (torch.min(y, 1)[0] - min_vals)
             max_vals = max_vals + self.averaging_constant * (torch.max(y, 1)[0] - max_vals)
-        self.min_vals = min_vals
-        self.max_vals = max_vals
+        self.min_vals.resize_(min_vals.shape)
```

Comment:
Will using the out variant help with speed? For example, can we use this:
torch.add(input, other, out=None)

Reply:
hmm, I took a look and it seems like only some flavors of `min/max` are supported with `=out`, so we wouldn't be able to apply that consistently throughout this diff.  Might make sense to keep it as is for simplicity, unless we are sure that this is a performance bottleneck. Thoughts/concerns?